assets/js/atomic/blocks/product-elements/sku/edit.tsx

```diff
@@ -18,7 +18,10 @@ const Edit = ( {
 	setAttributes,
 	context,
 }: BlockEditProps< Attributes > & { context: Context } ): JSX.Element => {
-	const blockProps = useBlockProps();
+	const { style, ...blockProps } = useBlockProps( {
+		className:
+			'wc-block-components-product-sku wp-block-woocommerce-product-sku',
```

Comment:
I apologize, but I am having trouble understanding your comment. Are you suggesting that `wc-block-components-product-sku` is equivalent to `wp-block-woocommerce-product-sku`? 🤔

Reply:
> Are you suggesting that wc-block-components-product-sku is equivalent to wp-block-woocommerce-product-sku? 🤔

Sorry I wasn't clear enough. What I mean is:

1. Here you provide a class name `wp-block-woocommerce-product-sku`, which is assigned to the `div` block container: [here](https://github.com/woocommerce/woocommerce-blocks/pull/8913/files#diff-273af84aa585a8e5a5a1446b8e97d85242eca59f5bf0696f3111b6627794ebaeR40)
2. The same class `wp-block-woocommerce-product-sku` is assigned to the Preview component [in here](https://github.com/woocommerce/woocommerce-blocks/pull/8913/files#diff-60f7e4bda641c87212e5a65b413e1c794870354e6431235268e5794f784c893fR80).

But I think I see what's happening, these are mutually exclusive, right? 🤔

Reply:
Thanks for the explanation. Now I can understand what you mean.

Actually, [here in block.tsx](https://github.com/woocommerce/woocommerce-blocks/blob/9e10039b4471566084ed3b99b808db92e3b713eb/assets/js/atomic/blocks/product-elements/sku/block.tsx#L77-L86), we are adding a class name for `All products`. Because classname added in [edit.tsx](https://github.com/woocommerce/woocommerce-blocks/blob/9e10039b4471566084ed3b99b808db92e3b713eb/assets/js/atomic/blocks/product-elements/sku/edit.tsx#L23) won't be added to `All products` blocks.

In summary, the class name in block.tsx will apply to the All products block, while the class name in edit.tsx will apply everywhere else. I hope that makes sense. 

Although I think this kind of handling shouldn't be required, but to support product elements in `All Products`, I added this 🙂